databases/week2/db1w2hw_hyf_lesson2_hwPart1-part3.sql

```diff
@@ -0,0 +1,122 @@
+-- Part 1: Working with tasks // DDL homework
+-- Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO
+  task
+VALUES
+  (
+    37,
+    'Designer',
+    'Database designer for startup company',
+    '2021-10-10 06:54:16',
+    '2022-10-10 13:05:09',
+    '2024-10-10 13:05:09',
+    1
+  );
+INSERT INTO
+  task (
+    id,
+    title,
+    description,
+    created,
+    updated,
+    due_date,
+    status_id
+  )
+VALUES
+  (
+    38,
+    'db designer',
+    'db sesigner for the first time.',
+    '2017-10-25 06:54:16',
+    '2017-10-15 13:05:09',
+    null,
+    2
+  );
+-- Change the title of a task
+UPDATE
+  task
+SET
+  title = 'sumans first job'
+WHERE
+  id = 36;
+-- Change a task due date
+UPDATE
+  task
+SET
+  due_date = '2022-02-08'
+WHERE
+  id = 36;
+-- Change a task status
+UPDATE
+  task
+SET
+  status_id = 3
+WHERE
+  id = 36;
+-- Mark a task as complete
+DELETE FROM
+  task
+WHERE
+  id = 38;
+-- Delete a task
+SELECT
+  *
+FROM
+  task;
+-- homework part 3 // DML homework
+  -- Get all the tasks assigned to users whose email ends in @spotify.com
+  USE hyf_lesson2;
+SELECT
+  task.title,
+  user.email
+FROM
+  task
+  join user_task ON task.id = user_task.task_id
+  JOIN user ON user_task.user_id = user.id
+WHERE
+  user.email LIKE "%@spotify.com";
+-- Get all the tasks for 'Donald Duck' with status 'Not started'
+SELECT
+  task.title,
+  user.name as user,
```

Comment:
it is correct, however its good to use a better alias like user_name, the VSCode plugin show wrong result if you dont use the alias at all, but thats not the case in any other IDE or terminal, just to let you know :)